extension-manifest-v3/src/manifest.chromium.json

```diff
@@ -125,6 +125,11 @@
       "matches": [
         "<all_urls>"
       ]
+    },
+    {
+      "resources": [
+        "rule_resources/statics/*"
```

Comment:
should be merged with the list above as it is meant to be used on all_frames and with <all_urls>

Reply:
We have a conflict with https://github.com/ghostery/ghostery-extension/blob/main/extension-manifest-v3/scripts/build.js#L219-L242